packages/schema/README.md

```diff
@@ -5432,6 +5432,19 @@ const result = S.decodeUnknownEither(Name)("").pipe(
 console.log(result); // => { _id: 'Either', _tag: 'Left', left: 'should be non empty' }
 ```
 
+# Comparison
+
+## Zod
+
+Feature-wise, `schema` can do practically everything that `zod` can do.
+
+The main differences are:
+
+1.  `schema` transformations are bidirectional, so it not only decodes like `zod` but also encodes.
+2.  `schema` is integrated with `Effect` and inherits some benefits from it (such as dependency tracking in transformations).
+3.  `schema` is highly customizable through annotations, allowing users to attach meta-information.
+4.  `schema` uses a functional programming style with combinators and transformations (while `zod` provides a chainable API).
```

Comment:
Avoid mentioning FP at any cost, IMHO 😅 

```suggestion
4.  `schema` provides many combinators and transformations making it highly composable (while `zod` provides a chainable API).
```

Reply:
But with my suggestion, I'm not sure how composability contrasts with the chainable API because you can compose stuff through chaining, too.

The downside is that chainable APIs are not tree-shakable because the methods are attached to the objects prototype itself. `schema` may result in much smaller bundle size compared to `zod`, which is something frontenders will appreciate. But this needs to be verified in practice, I haven't done that.

Reply:
Given that schema will include the Effect runtime the smaller bundle size will only materialize in extreme use cases,  wouldn't make such argument. What we can safely say is that chainable APIs can't be extended due to lack of extension functions in JS / TS, pipeable APIs can be extended as they are just composition of functions